types/index.d.ts

```diff
@@ -1,12 +1,19 @@
-import { FastifyMiddleware } from 'fastify'
+import * as fastify from 'fastify'
+import * as http from 'http'
 
-export interface HealthcheckPluginOptions {
+export interface FastifyHealthcheckOptions {
   healthcheckUrl?: string
   healthcheckUrlDisable?: boolean
   healthcheckUrlAlwaysFail?: boolean
   exposeUptime?: boolean
   underPressureOptions?: Record<string, any>
 }
 
-declare const healthcheckPlugin: FastifyMiddleware<HealthcheckPluginOptions>
-export default healthcheckPlugin
+export const fastifyHealthcheck: fastify.Plugin<
```

Comment:
Renamed to be more consistent with other fastify plugin naming

Reply:
very good